content/interfaces/cli.md

```diff
@@ -218,25 +218,25 @@ Runs a SQL query
 ### Synopsis
 
 ```bash
+dolt sql 
+dolt sql < script.sql
 dolt sql [--multi-db-dir &lt;directory&gt;] [-r &lt;result format&gt;]
-dolt sql -q &lt;query;query&gt; [-r &lt;result format&gt;] -s &lt;name&gt; -m &lt;message&gt; [-b] [&lt;commit&gt;]
-dolt sql -q &lt;query;query&gt; --multi-db-dir &lt;directory&gt; [-r &lt;result format&gt;] [-b]
-dolt sql -x &lt;name&gt; [&lt;commit&gt;]
+dolt sql -q &lt;query&gt; [-r &lt;result format&gt;] [-s &lt;name&gt; -m &lt;message&gt;] [-b]
+dolt sql -q &lt;query&gt; --multi-db-dir &lt;directory&gt; [-r &lt;result format&gt;] [-b]
+dolt sql -x &lt;name&gt;
 dolt sql --list-saved
 ```
 
 ### Description
 
-Runs a SQL query you specify. With no arguments, begins an interactive shell to run queries and view the results. With the `-q` option, runs the given query and prints any results, then exits. If a commit is specified then only read queries are supported, and will run against the data at the specified commit.
+Runs a SQL query you specify. With no arguments, begins an interactive shell to run queries and view the results. With the `-q` option, runs the given query and prints any results, then exits.
 
 By default, `-q` executes a single statement. To execute multiple SQL statements separated by semicolons, use `-b` to enable batch mode. Queries can be saved with `-s`. Alternatively `-x` can be used to execute a saved query by name. Pipe SQL statements to dolt sql (no `-q`) to execute a SQL import or update script. 
 
-By default this command uses the dolt data repository in the current working directory as the one and only database. Running with `--multi-db-dir <directory>` uses each of the subdirectories of the supplied directory (each subdirectory must be a valid dolt data repository) as databases. Subdirectories starting with '.' are ignored.
+By default this command uses the dolt data repository in the current working directory, as well as any dolt databases that are found in the current directory. Any databases created are placed in the current directory as well. Running with `--multi-db-dir <directory>` uses each of the subdirectories of the supplied directory (each subdirectory must be a valid dolt data repository) as databases. Subdirectories starting with '.' are ignored.
```

Comment:
"Any databases created" could use more context. As in "created with `CREATE DATABASE` commands" or "created via DDL..."